app/livechat/server/hooks/leadCapture.ts

```diff
@@ -28,15 +42,15 @@ function validateMessage(message, room) {
 
 callbacks.add(
 	'afterSaveMessage',
-	function (message, room) {
+	function (message: MessageData, room: any): MessageData {
```

Comment:
Same here

Reply:
done in d49bf2abb7451df60b635c0fcc8f2c85ee745617 too